addons/account/models/partner.py

```diff
@@ -105,7 +105,7 @@ def map_tax(self, taxes):
             return taxes
         result = self.env['account.tax']
         for tax in taxes:
-            taxes_correspondance = self.tax_ids.filtered(lambda t: t.tax_src_id == tax._origin)
+            taxes_correspondance = self.tax_ids.filtered(lambda t: t.tax_src_id == tax._origin and t.tax_dest_id.active)
```

Comment:
```suggestion
            taxes_correspondance = self.tax_ids.filtered(lambda t: t.tax_src_id == tax._origin and t.tax_dest_id_active)
```
I guess you could use the new field since you created it for the view?